### PR TITLE
Tie break feature

### PR DESCRIPTION
**Pull request by ywrsusan**

### CODE REVIEW

File: run_tests [hunk 1]
```diff
@@ -1,7 +1,7 @@
 #!/bin/bash
-
+set -x
 this_dir="$(cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd)"
 
 PYBOSSA_SETTINGS="${this_dir}/settings_test.py" \
     PYBOSSA_REDIS_CACHE_DISABLED=1 \
-    nosetests ${@:-test/}
+    nosetests -s ${@:-test/}
```
Comment (fiorda) [inline run_tests]: can we remove `-s`? We can't see the whole test output in travis anymore

Comment (ywrsusan) [reply run_tests]: Sure we can remove that.

File: pybossa/view/projects.py [hunk 2]
```diff
@@ -2992,21 +2991,32 @@ def answerfieldsconfig(short_name):
 
     if request.method == 'POST':
         try:
-            project.info['answer_fields'] = json.loads(request.data)
+            body = json.loads(request.data) or {}
+            answerFieldsConfig = body.get('answerFieldsConfig') or {}
+            consensusConfig = body.get('consensusConfig') or {}
+            if answerFieldsConfig:
```
Comment (jayflo) [inline pybossa/view/projects.py]: Looking at this now, it seems that prior to this change, there was

```
project.info['answer_fields'] = json.loads(request.data)
```

which didn't care if `answerFieldsConfig` was empty (i.e. `project.info['answer_fields']` could be "blanked out").  I'm not sure if this is needed or if `answerFieldsConfig` can be removed in another way.

Comment (ywrsusan) [reply pybossa/view/projects.py]: true I fix it in latest commit

**Pull request merged**